storage/statedb/cache_hybrid_test.go

```diff
@@ -27,6 +28,7 @@ func TestHybridCache_Set(t *testing.T) {
 	// Set an item
 	key, value := randBytes(32), randBytes(500)
 	cache.Set(key, value)
+	time.Sleep(100 * time.Millisecond)
```

Comment:
How about setting the sleep time with `redisCacheTimeout`?
```suggestion
	time.Sleep(redisCacheTimeout)
```

Reply:
Thank you for the interesting suggestion, but I am trying to increase `redisCacheTimeout` in https://github.com/klaytn/klaytn/pull/777. The value could be longer than currently suggested if this async set PR is merged. I think it is a little long to use frequently.

Reply:
1. `100 * time.Millisecond` looks fine for now.
2. I think there should be a vairable for the time, though. It is used a lot in the redis test codes.

Reply:
Converted it to a variable, `sleepDurationForAsync`

Reply:
Thank you for adopting the suggestion :)